### DISCUSSION TITLE
post/school-songs/

### DISCUSSION THREAD

**Discussion by giscus** [SHIERTX/static-blog#4] [Comments]
# post/school-songs/

分享下学校的两首歌吧，HCZX YYDS！
合川中学，我将是你的骄傲 由初2022级——也是我们班的语文老师福中作词，蒋仕杰作曲的 《合川中学，我将是你的骄傲》 很受大家欢迎[1]，没几天就飞升到了我网易云音乐播放次数第二名。
原曲 站内播放  网易云音乐官方外链播放  网易云音乐歌曲地址 https://music.163.com/#/song?id=1858499114
伴奏 站内播放  网易云音乐官方外链播放  网易云音乐歌曲地址 https://music.163.com/#/song?id=1876293111
登上合中大台阶 也是福中作词，开幕雷击，播放前请做好心理准备:)
原曲 站内播放  网易云音乐官方外链播放  网易云音乐歌曲地址 https://music.163.com/#/song?id=1858499120
伴奏 站内播放  网易云音乐官方外链播放  网易云音乐歌曲地址 https://music.163.com/#/song?id=1876293537
相关资源 合川中学原创歌曲资源包 直链下载：点我
天翼云盘：传送门
引用内容 [1] : 数据来源网易音乐人，截图

https://www.liczhi.com/post/school-songs/

**Comment by cheezhi** [comment]
十二有QQ吗

**Reply by cheezhi** [reply]
QQ号是多少啊

**Reply by zaochih** [reply]
2271897521

**Comment by lizhijunya** [comment]
羡慕了 我们学校连个屁都没有
~但是这歌有点那啥怪怪的~

**Reply by zaochih** [reply]
俺也觉得（）

**Reply by cheezhi** [reply]
我觉得挺好听的
